src/components/appNavigation/TabsView.js

```diff
@@ -86,7 +86,7 @@ const RewardButton = React.memo(({ onPress, style }) => {
     <>
       <TouchableOpacity testID="rewards_tab" onPress={onPress} style={style}>
         <Icon name="rewards" size={36} color="white" />
-        {updatesCount > 0 && (
+        {updatesCount > -1 && (
```

Comment:
why this is needed ? QA reported about broken layout only.
does we really need to display zero value ?

```suggestion
        {updatesCount > 0 && (
```